executor/join.go

```diff
@@ -71,6 +72,17 @@ type HashJoinExec struct {
 	hashTableValBufs   [][][]byte
 
 	memTracker *memory.Tracker // track memory usage.
+
+	// useRadixPartition indicates whether to use radix-based cluster algorithm
+	// to partitioning.
+	// Note: Only used for testing now, we'll check whether to use radix
+	// partition according to the the skew of inner data when radix partition is
+	// totally supported.
+	UseRadixPartition bool
```

Comment:
s/partitioning/partition
s/totally/fully

Reply:
I think we only need `radixBits` to represent whether to enable the radix-based partition algorithm:
- `radixBits == 0`, the radix hash join is disabled.
- `radixBits >= 1`, the radix hash join is enabled.